python-runtime/ftl/src/ftl/extract/context.py

```diff
@@ -0,0 +1,95 @@
+import importlib.util
+import multiprocessing
+import threading
+
+from ftl.protos.xyz.block.ftl.v1.schema import schema_pb2 as schemapb
+
+"""Global context across multiple files in a package."""
```

Comment:
Is this meant to be attached to the class below?

Reply:
aye done